packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -266,21 +281,31 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
         if (_ident == 1) {
             // Load the L1 head hash into the game's local context in the preimage oracle.
             oracle.loadLocalData(_ident, Hash.unwrap(l1Head), 32, _partOffset);
-        } else if (_ident == 2) {
-            // Load the earliest output root that commits to the passed L2 block number
-            // into the game's local context in the preimage oracle.
-            Types.OutputProposal memory proposal = L2_OUTPUT_ORACLE.getL2OutputAfter(
-                l2BlockNumber()
+        } else if (_ident < 4) {
+            // Grab the index of the proposal to load at the given ident.
+            // Ident 2 loads the starting output root, and ident 3 loads the disputed output root.
+            uint256 proposal;
+            assembly {
+                proposal := iszero(mod(_ident, 2))
```

Comment:
nit: We can avoid using assembly here if the proposals were separated out rather than stored in an array